test/e2e/e2e.go

```diff
@@ -130,39 +128,6 @@ func RunE2ETests(t *testing.T) {
 	ginkgo.RunSpecsWithDefaultAndCustomReporters(t, "Kubernetes e2e suite", r)
 }
 
-// Run a test container to try and contact the Kubernetes api-server from a pod, wait for it
-// to flip to Ready, log its output and delete it.
-func runKubernetesServiceTestContainer(c clientset.Interface, ns string) {
-	path := "test/images/clusterapi-tester/pod.yaml"
```

Comment:
did some archeology and this function was added here:
https://github.com/kubernetes/kubernetes/commit/33bb8e863cae888b37a8ffbdbcd2a6db19a156b8

it's not related to the Cluster API project, but rather it's a generic k8s cluster API tester of sorts.
following the git blame was interesting, since it was moved between (the same) files at least 4 times.

+1 for removal if unused.

Reply:
https://github.com/kubernetes/kubernetes/blob/ab609a418ea804c896cbcba4d93a77e46ca76dff/test/images/Makefile#L32

we should update this example too.

Reply:
done